### PR TITLE
updated prefabs

### PR DESCRIPTION
**Pull request by Charles-Huang-Y4**

### CODE REVIEW

File: Assets/Scripts/Hazards/Run.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using UnityEngine;
+
+public class Run : MonoBehaviour { 
+    [SerializeField] private GameObject _blobManager;
+    [SerializeField] private GameObject _shooter;
+    [SerializeField ]private AudioSource _audioSource;
```
Comment (AlexisCM) [inline Assets/Scripts/Hazards/Run.cs:7]: Fix spacing surrounding square bracket

File: Assets/Scripts/Hazards/Run.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    [SerializeField] private float _trackStartTime;
+    [SerializeField] private float _time;
+    void Start()
+    {
```
Comment (AlexisCM) [inline Assets/Scripts/Hazards/Run.cs:12]: No opening brace on the newline, should be on the same line as `Start()`

**Review by AlexisCM — CHANGES_REQUESTED** [review]

**Review by AlexisCM — APPROVED** [review]
LGTM~

**Comment by mbajaman** [comment]
TIL that lgtm is an acronym

**Comment by AlexisCM** [comment]
TIL that TIL is an acronym

**Pull request merged**